### PR TITLE
Add GPS coordinates to GT and CSV export

### LINKED ISSUE #83: Image EXIF in ground truth and export
## Feature description and context
Building on #82 , we should provide the EXIF GPS to lab leads during analysis, both during the review phase and during external analysis.

## Feature sign-off requirements
Export Image csv format includes new columns after **Image Filename:**
- **Date Exif**: If available, the date pulled from the image's exif
- **GPS Lat Exif**: If available, the latitude pulled from the image's exif, format decimal degrees
- **GPS Long Exif**: If available, the longitude pulled from the image's exif, format decimal degrees

In Ground Truth page, display date, latitude and longitude (if available) as overview information about the matching image

## Design
_Note: not pixel perfect representation, just general UX. Leverage existing patterns/styles_
image

### PR DESCRIPTION
**Pull request by ejcunningham**
Parse GPS from EXIF data and plumb it through to the UI and CSV exporter

PR fixes #83

**Changes**
- Add GPS parser function to grab exif data and convert to decimal
- Add lat+long to the Images model
- Add a small metadata section to the GT sidebar with date, lat, long (or 'unavailable')
- Add lat, long, and date to the CSV export

Note the ticket specifies putting the metadata in the upper left corner, but I thought this information would be too crowded up there. Open to suggestions for how to format this.

Screenshots:
New CSV export:
Screenshot 2024-06-01 at 3 18 50 PM

Image with location data:
Screenshot 2024-06-01 at 3 44 43 PM

Image without location data:
Screenshot 2024-06-01 at 3 17 51 PM

### CODE REVIEW

**Review by TanyaStere42 — APPROVED** [review]
UX review only: that design choice works for me, information is grouped and legible while not distracting. Approved pending code review from @naknomum

File: app.js
Comment (naknomum) [inline app.js:1420]: i think it would be best to wrap some of this a `try... catch` or similar, just to handle the case where unexpected exif data is parsed from the image. for example, catching where `numerator` and/or `denominator` are undefined or cause divide-by-zero error. or when `parts.length < 3` etc.

**Review by naknomum — CHANGES_REQUESTED** [review]
just one comment about handling unexpected exif values.

File: app.js
Comment (ejcunningham) [reply app.js:1421]: Very good point, this code makes a lot of assumptions about consistent formatting. Updated with improved error handling.

**Review by naknomum — APPROVED** [review]
looks good

**Pull request merged**
